test/fixtures/dialects/ansi/with_nested_in_with_statement.sql

```diff
@@ -1,5 +1,4 @@
-(
-    WITH mycte2 AS (
+(WITH mycte2 AS (
```

Comment:
L053 fixes this by removing the brackets which I guess leads to `Whitespace(), Whitespace()` which is unparsable (I think) - Fixing this bug which is in L053 seemed out of scope so I just removed the padding to allow this case to pass for now.

Reply:
Can you capture enough detail and create an issue about L053? Sounds like something that should be addressed soon-ish.

Reply:
BTW, the PR I merged today included some L053 fixes. Is it possible I already addressed the problem you were seeing?

Reply:
Yea loads of info, my edit to the cirtical errors test suite gives a detailed desc of "valid" ---> "invalid"